src/second-into-iter.md

```diff
@@ -31,7 +31,7 @@ iterator each collection should endeavour to implement:
 
 We actually already have all the tools to implement
 IntoIter using List's interface: just call `pop` over and over. As such, we'll
-just implement IntoIter as a newtype wrapper around List:
+just implement IntoIter as a new type wrapper around List:
```

Comment:
```suggestion
just implement IntoIter as a newtype wrapper around List:
```

This one is a Term Of Art, not a typo

Reply:
Oh cool! Searching for it yielded only `new type` but I probably should have tried quoting it! I'll remove this from the commit.

Reply:
https://rust-unofficial.github.io/patterns/patterns/behavioural/newtype.html